src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionCommentFilterTest.java

```diff
@@ -422,9 +418,26 @@ public void testTagsAreClearedEachRun() {
         final TreeWalkerAuditEvent dummyEvent2 = new TreeWalkerAuditEvent(contents2, "filename",
                 new LocalizedMessage(1, null, null, null, null, Object.class, null), null);
         suppressionCommentFilter.accept(dummyEvent2);
-        final List<SuppressionCommentFilter.Tag> tags =
-                Whitebox.getInternalState(suppressionCommentFilter, "tags");
+        final List tags = Whitebox.getInternalState(suppressionCommentFilter, "tags");
```

Comment:
why it is different from https://github.com/checkstyle/checkstyle/pull/6855/commits/f9f08c12602fda12aba4e65df63cdd95bad53437#diff-ee5ff681a2b960c09977ac668764e675R473

Reply:
I'm surprised no CI flagged the bare `List` and requested `List<...>`.